solr/solrj-zookeeper/src/java/org/apache/solr/common/cloud/ClusterProperties.java

```diff
@@ -220,7 +220,7 @@ public void setClusterProperty(String propertyName, Object propertyValue) throws
           @SuppressWarnings({"rawtypes"})
           Map properties = new LinkedHashMap();
           properties.put(propertyName, propertyValue);
-          client.create(
+          client.makePath(
```

Comment:
minor: this doesn't seem to need the change, given that it's a constant.

Reply:
This is what necessitated the change. the `chroot` that the user is using might not exist yet.

Reply:
yep. wrote the comment and like 10 mins later...oh yeah... thanks for confirming :)